pages/common/avo.md

```diff
@@ -0,0 +1,36 @@
+# avo
+
+> The official command-line interface for Avo.
+> More information: <https://www.avo.app/docs/implementation/cli>.
+
+- Initialize a workspace in the current directory:
+
+`avo init`
+
+- Log into the Avo platform:
+
+`avo login`
+
+- Switch to an existing Avo branch:
+
+`avo checkout {{branch_name}}`
+
+- Pull analytics wrappers for the current path:
+
+`avo pull`
+
+- Display the status of the Avo implementation:
```

Comment:
I dont like "implementation"

I dont think it is the correct usage

Reply:
That's what the help page says. Because the output is which avo log event is not implemented yet, or where is used ("implemented") each Avo log event function.

Reply:
What about "display the status of Avo

Reply:
> That's what the help page says. Because the output is which avo log event is not implemented yet, or where is used ("implemented") each Avo log event function.

In this case I think `implementation` should be fine.